### PR TITLE
Evaluation should use validation datasets

### PR DESCRIPTION
**Pull request by Akelio-zhang**
Evaluation should use 'validation' (not 'train' datasets) datasets in the last part (Apply fine tuned model to some images.) of this walkthrough file.

### CODE REVIEW

**Comment by Akelio-zhang** [comment]
I signed it!

**Comment by Akelio-zhang** [comment]
I just signed it !

**Comment by zongyuange** [comment]
@Akelio-zhang After you switched from 'train' to 'validation', also `logits,_ = inception.inception_v1(images, num_classes=dataset.num_classes, is_training=True)`, to
`logits,_ = inception.inception_v1(images, num_classes=dataset.num_classes, is_training=False)`,
did you see a significant accuracy drop?

**Comment by Akelio-zhang** [comment]
@zongyuange Yes, I tried this code and the result was like your words, but I am not sure if this configuration is reasonable during evaluation.

**Comment by Akelio-zhang** [comment]
@zongyuange Yes, I tested this code and the result was like your words. Besides, [here](https://github.com/tensorflow/models/issues/391#issuecomment-247392028) discussed `is_training`'s effect

**Comment by zongyuange** [comment]
@Akelio-zhang, thanks for your comment, that's really helpful. 
I think TF-Slim should also update their fine-tuning example incase any other confusion may raise in the future.
